packages/web-service/src/pages/returns/common-return-functions.js

```diff
@@ -119,3 +121,18 @@ export const getWhyNoArtificialSettReason = returnData => {
   }
   return whyNoArtificialSettReason
 }
+
+export const allCompletion = async (request) => {
+  const journeyData = await request.cache().getData()
+
+  const returnId = journeyData?.returns?.id
```

Comment:
if .returns undefined is possible we should have a check function throw the user back to the applications page and expect .returns defined

Reply:
Totally. Will add this 👍